ddtrace/contrib/unittest/patch.py

```diff
@@ -33,64 +50,253 @@ def get_version():
     return ""
 
 
-def _set_tracer(tracer):
+def _set_tracer(tracer: ddtrace.tracer):
+    """Manually sets the tracer instance to `unittest.`"""
     unittest._datadog_tracer = tracer
 
 
-def _store_span(item, span):
-    """Store span at `unittest` instance."""
+def _store_test_span(item, span: ddtrace.Span):
+    """Store datadog span at `unittest` test instance."""
     item._datadog_span = span
 
 
-def _extract_span(item):
-    """Extract span from `unittest` instance."""
+def _store_module_identifier(test_object: unittest.TextTestRunner):
```

Comment:
The 2.x branch dropped support for python2 so I guess this works. What I had in mind was something like [this](https://github.com/DataDog/dd-trace-py/blob/c6808b2d6d016aa600b7c52c50b172eeca44bd0d/ddtrace/internal/sma.py#L39). Using type hints in comments better aligns with how we currently use mypy in ddtrace. 

But this works. I think we should move towards using python3 syntax anyway.